docs/src/data/tailwind-migration-status.yaml

```diff
@@ -79,7 +79,7 @@ Skeleton: true
 SkipLink: false
 SkipNavigation: false
 Slide: false
```

Comment:
What’s `Slide`? Is it a typo?

Reply:
It's a util that has styled-components.

https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/utils/Slide/index.tsx